### PR TITLE
Updates for the latest DASHSchema

### PR DESCRIPTION
**Pull request by minhkstn**
Contributor: Daniele Lorenzi
DASHSchema version: 5th-Ed

### CODE REVIEW

File: libdash/libdash/source/manager/DASHManager.cpp [hunk 1]
```diff
@@ -34,6 +38,203 @@ IMPD*           DASHManager::Open   (char *path)
 
     MPD* mpd = parser.GetRootNode()->ToMPD();
 
+    std::cout << "Variables check:" << std::endl;
```
Comment (msmole) [inline libdash/libdash/source/manager/DASHManager.cpp:41]: That looks like some testing code that should not be part of the PR, right?
If so, please remove that whole part.

**Review by msmole — CHANGES_REQUESTED** [review]

File: libdash/libdash/README_mpd.txt [hunk 2]
```diff
@@ -0,0 +1,12 @@
+DASH-IF/Github repository - New Elements Test:
```
Comment (msmole) [inline libdash/libdash/README_mpd.txt:1]: What value does this new file provide? Seems there is a status "working" noted on all the added manifests. What does that mean? How was that tested?

If you think this files is not that important then just delete it. If you think it is important, I think it would be worth to add some more description to it and transform it to the MD format. Probably also good to link it from the root level README.md file?

**Review by msmole — CHANGES_REQUESTED** [review]

File: libdash/libdash/source/manager/DASHManager.cpp (hunk 1 above)
Comment (zenzi-deluxe) [reply libdash/libdash/source/manager/DASHManager.cpp]: You are right. It was part of the testing code. I have already provided to the code removal.

File: libdash/libdash/README_mpd.txt (hunk 2 above)
Comment (zenzi-deluxe) [reply libdash/libdash/README_mpd.txt]: This file had been created with the intent of keeping trace of the evaluation and test of the new added elements - together with the tested MPD file - but it is not so relevant anymore to the update itself by the time every new added element is working as expected. I have already provided to the file removal.

**Review by msmole — APPROVED** [review]

**Review by bitressi — APPROVED** [review]
